args/linkedin.go

```diff
@@ -10,6 +18,72 @@ type LinkedInArguments struct {
 	Start            int      `json:"start"`                     // Pagination start offset
 }
 
+// UnmarshalJSON implements custom JSON unmarshaling with validation
+func (l *LinkedInArguments) UnmarshalJSON(data []byte) error {
+	type Alias LinkedInArguments
+	aux := &struct {
+		*Alias
+	}{
+		Alias: (*Alias)(l),
+	}
+
+	if err := json.Unmarshal(data, aux); err != nil {
+		return fmt.Errorf("failed to unmarshal LinkedIn arguments: %w", err)
+	}
```

Comment:
Why do we need all of this? Wouldn't `json.Unmarshal(data, l)` work? Inquiring minds want to know.